### ISSUE TITLE
Map thumbnails url are only relative

### ISSUE THREAD

**Issue by Gnafu** [geosolutions-it/MapStore2#1500]
The map thumbnails are stored in the relative form:
/mapstore/rest/geostore/data/1744/raw?decode=datauri

So if the map is loaded from another domain it will not have a visible thumbnail.

The thumbnails should be stored with the full URL:
https://mapstore2.geo-solutions.it/mapstore/rest/geostore/data/1744/raw?decode=datauri

**Comment by offtherailz** [comment]
I don't think the map's should be saved as absolute. Instead they should be relative to the geostore instance. Otherwise domain names changes and database copies invalidate the thumbnail URL. 
I provided an old fix here 353353638a173fec9d2d300e6b013ee22e245806 but I don't know if it is valid yet. 
The fix I provided should work as I said, keeping relative URLs and using geostore as a base, but I don't remember if it was a work in progress and why we didn't merge it, it was 8 month ago. 
As far as I know, this fix may also be applied during one of the CSW review.

I should test this before
